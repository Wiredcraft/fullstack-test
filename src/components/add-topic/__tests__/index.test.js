import React from 'react';
import renderer from 'react-test-renderer';

import AddTopic from '../index';

jest.mock('react-router-dom');

test('AddTopic renders', () => {
  const component = renderer.create(<AddTopic />);

  expect(component.toJSON()).toMatchSnapshot();
});
