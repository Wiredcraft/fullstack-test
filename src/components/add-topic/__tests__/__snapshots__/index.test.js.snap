// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddTopic renders 1`] = `
<form
  onSubmit={[Function]}
>
  <div
    className="sc-bdvvtL jktTcy"
  >
    <span
      className="sc-gsDKAQ hvZYDJ"
    >
      Submit your lightning talk!
    </span>
    <input
      className="sc-dkPtRN ejKAfo"
      name="title"
      onChange={[Function]}
      placeholder="Title"
      type="text"
      value=""
    />
    <textarea
      className="sc-hKwDye dvofbG"
      name="description"
      onChange={[Function]}
      placeholder="Describe your talk"
      type="text"
      value=""
    />
    <input
      className="sc-dkPtRN ejKAfo"
      name="user"
      onChange={[Function]}
      placeholder="Your name"
      type="text"
      value=""
    />
    <button
      className="sc-eCImPb lbvQNE"
      disabled={true}
      onSubmit={[Function]}
      type="submit"
    >
      Submit
    </button>
  </div>
</form>
`;
